src/components/helpers/ajax.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios'
+
+class Ajax {
+  fetchTags () {
+    return axios.get('http://localhost:3001/api/v1/tags.json')
```

Comment:
on that note, I like to separate url strings outside of the fetching method, this way in the future you can just change it there, instead of inside the actual get. Extremely minor and entirely not needed